src/new-components/Faq.vue

```diff
@@ -0,0 +1,322 @@
+      <div class="categories">
+        <div
+          v-for="name in Object.keys(faqData)"
+          :key="name"
+          :class="{ 'category-bubble': true, selected: selected === name }"
+          
+        >
+          <div class="category-name" @click="selectCategory(name)">
+            {{
+            name.charAt(0).toUpperCase() + name.substring(1)
+            }}
+          </div>
+          <transition name="open">
+            <div :key="name + 's'" v-if="selected === name" class="mobile-faq-area">
+              <div
+                v-for="(elm, index) in faqData[selected]"
+                :key="elm.q"
+                @click="toggleQuestion(index)"
+                class="question-box"
+              >
+                <div class="question">
+                  {{ elm.q }}
+                  <span :class="{ arrow: true, rotate: index === expanded }">
+                    <i v-if="expanded.includes(index)" class="fa fa-caret-up fa-xs"></i>
+                    <i v-else class="fa fa-caret-down fa-xs"></i>
+                  </span>
+                </div>
+                <transition name="open">
+                  <div v-if="expanded.includes(index)" class="answer-box">
+                    {{
+                    elm.a
+                    }}
+                  </div>
+                </transition>
+              </div>
+            </div>
+          </transition>
+        </div>
+      </div>
+      <div class="faq-area">
+        <div v-for="(elm, index) in faqData[selected]" :key="elm.q" class="question-box">
+          <div class="question" @click="toggleQuestion(index)">
+            {{ elm.q }}
+            <span :class="{ arrow: true }">
+              <i v-if="expanded.includes(index)" class="fa fa-caret-up fa-xs"></i>
+              <i v-else class="fa fa-caret-down fa-xs"></i>
+            </span>
+          </div>
+          <transition name="open">
+            <div v-if="expanded.includes(index)" class="answer-box">
+              {{
+              elm.a
+              }}
+            </div>
+          </transition>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import faqData from '@/data/faq_data.ts';
+
+export default Vue.extend({
+  name: 'Faq',
+  props: ['initialSelect'],
+  data(): { faqData: any; selected: string; expanded: number[] } {
+    return {
+      faqData,
+      selected: '',
+      expanded: [0],
+    };
+  },
+  methods: {
+    isMobile: (): boolean => window.innerWidth <= 700,
+    toggleQuestion(index: number) {
+      if (!this.expanded.includes(index)) {
+        this.expanded.push(index);
+      } else {
+        this.expanded = this.expanded.filter((each) => each !== index);
+      }
+    },
+    selectCategory(name: string) {
+      if(this.selected !== name){
+        this.selected = name;
+        this.expanded = [0];
+      }
+      else {
+        if(this.isMobile()){
+          this.selected='';
```

Comment:
(Non-blocker) mind looking at the warnings the linter's throwing out when you run `yarn run serve` related to these lines?

Reply:
Yeah I took a look, should be all good now!